README.md

```diff
@@ -1,35 +1,46 @@
 
 ### Time Required: 1 hour
   - 10 mins to read challenge instructions
   - 40 mins to complete the challenge in pairs
   - 10 mins to regroup and talk over the solution
 
 ### Learning outcome
-To become familiar with the Request module and the Node core [http module](https://nodejs.org/api/http.html).
+To become familiar with making API calls from the backend using the Node core [http module](https://nodejs.org/api/http.html).
 
-### What is the Node Request module?
-Request is a very popular module that helps to make HTTP requests as simple as possible and it is built around the Node HTTP core module. [Here](http://stackabuse.com/the-node-js-request-module/) is a brilliant article on the Request module.
+### When would I need to make an API call from the backend?
+Example 1 - I would like to make a HTTP request to the [GitHub API](https://developer.github.com/v3/) to get my profile photo URL.
 
-### When would I need to use the request module?
-Example 1 - I would like to GET my profile-photo.png from the database to display on my homepage.
+Example 2 - I would like to make a HTTP request to the [TFL API](https://api.tfl.gov.uk/), to find out which buses go between London Bridge and Finsbury Park.
 
-Example 2 - I would like to use the Request module to make a HTTP request to the [TFL API](https://api.tfl.gov.uk/), to find out which busses go between London Bridge and Finsbury Park.
+### The challenge
 
-### Your challenge
-Create a function called **myRequest** that has similar set of features to the Node Request module.
+Create a function called `myRequest` which makes a request to an API url using an http GET request.
 
-The npm Request module simply creates a wrapper around the Node core http module. Your task it to create your own request function `myRequest` that performs in the same way as the Request npm module, which is used in the app.js file in this repo.
+It should take two arguments: the url to make the request to, and a callback function.
+
+The callback function will have two arguments:
+
+  1. error (String: if an error occurred)
+  2. response (Object: includes the body & statusCode of the request)
 
 **Hint**: relate to your knowledge of streams and sending data in chunks.
 
 To complete this challenge you will need to use the [`http.get`](https://nodejs.org/api/http.html#http_http_get_options_callback) method of the node core `http` module.
 
 ### Start here!
 1) Clone this repo & run `npm install`
-2) Run the app.js file in your command line with the command `node app.js`. This is the response from the request module that you're trying to recreate with your myRequest function. There is no need to change the helper function code.
-3) Open app.js and follow the instructions to create your own request function. 
+2) Run `npm test` in your command line to start the test watcher. The tests should fail as your function doesn't exist yet
+3) Open `app.js` and write your function!
+
+### Tests
```

Comment:
Not sure if this section will be helpful or just confusing. They can always read the tests if the instructions above aren't clear enough

Reply:
When @bantek89 tried this out for practice I think she found the tests useful?

Reply:
I agree that the tests are useful, I just don't know if a section explaining what the tests do adds much. If the tests themselves are confusing then maybe they should have comments next to them in the code?

Reply:
OK I've removed them